infrastructure/docker-images/build.sh

```diff
@@ -15,3 +17,14 @@ RUN apt-get update && apt-get install -y firefox-esr xvfb
 EOF
 
 docker build -t "taskcluster/browser-test:${node_version}" ${tmpdir}
+
+# Next we do the rabbit-test image
+
+tmpdir=$(mktemp -d)
+trap "cd /; rm -rf ${tmpdir}" EXIT
+cat > ${tmpdir}/Dockerfile <<EOF
+FROM node:${node_version}-stretch
+RUN apt-get update && apt-get install -y rabbitmq-server
```

Comment:
I wonder if we'll run into trouble with RabbitMQ versions here.  We can adjust if that happens..

Reply:
Yeah, I figure let's do the easiest thing now and build this up to be more complete later.